bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/gmos/GmosScienceAreaGeometry.scala

```diff
@@ -13,12 +13,12 @@ class GmosScienceAreaGeometry[I  <: InstGmosCommon[D,F,P,SM],
                               D  <: Enum[D]  with GmosCommonType.Disperser,
                               F  <: Enum[F]  with GmosCommonType.Filter,
                               P  <: Enum[P]  with GmosCommonType.FPUnit,
-                              SM <: Enum[SM] with GmosCommonType.StageMode](inst0: I) extends ScienceAreaGeometry[I] {
+                              SM <: Enum[SM] with GmosCommonType.StageMode](inst0: I) extends ScienceAreaGeometry {
   import GmosScienceAreaGeometry._
 
   override def geometry: List[Shape] = {
     Option(inst0).fold(List[Shape]()) { inst =>
-      lazy val width   = inst.getScienceArea()(0)
+      lazy val width   = inst.getScienceArea()(1)
```

Comment:
Note that this was a bug that Bryan pointed out to me. I should have been using height (idx 1) instead of width (idx 0).